style/javascript.md

```diff
@@ -536,6 +537,404 @@ to replace all uses of `var` in existing files.
 
 
 -----------------
+### Move off Underscore
+
+Now that we're using ES6/7 many of the features of Underscore.js are now built in to the language! We should move away from using Underscore in our code towards making use of these native features.
+
+There are a couple methods that are sufficiently complicated and don't have a direct equivalent so instead we have a custom-built copy of lodash containing only those methods. You can find this file at: `third_party/javascript-khansrc/lodash/lodash.js` along with instructions on how to build it and exactly what methods are included.
+
+What follows is a method-by-method set of equivalents for what we've used in Underscore and what you should be using in ES6/7 instead:
```

Comment:
I agree with the other comment that this would be more readable as a table instead of a doc.  I tried to read this section but all the changes in font size and location (due to the mix of code blocks and text blocks) made for a really jarring reading experience.  It was also really hard for me to see all the rules at a glance.

I would suggest that you have a table that covers the simple cases, and then individual sub-sections that cover some of the subtleties and special cases.  This is what I did for the ES6/7 section: the main table, plus more detailed discussion for bind and backticks.

I think you can make some of these examples smaller too, without loss of clarity.  For instance:

```
_.delay(function() {
    loadJavaScript();
    requireSuggestions();
}, 2000);
```

I think would also work as well as:

`_.delay(function() { ... }, 2000);`